### PR TITLE
My Acclaro

### PR DESCRIPTION
**Pull request by msanz-acclaro**
---
#### Important: Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image

image

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
@msanz-acclaro thank you for your submission. We look forward to getting your connector certified.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Comment by Amjed-Ayoub** [comment]
Hi @msanz-acclaro Thank you for reaching out. Please expect to hear back from our Certification Engineer at the beginning of the next week. 
Thank you for your patience.
Have a great weekend.

File: custom-connectors/My Acclaro/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# My Acclaro
```
Comment (sriyen-msft) [inline custom-connectors/My Acclaro/README.md:1]: @msanz-acclaro You have placed your connector under "custom-connectors" which is not going to be deployed and published into any production region. 

Docs - https://github.com/microsoft/PowerPlatformConnectors#microsoft-power-platform-connectors

File: custom-connectors/My Acclaro/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,13329 @@
...
+                      "type":"string",
+                      "description":"version"
+                    },
+                    "endpoint":{
+                      "type":"string",
+                      "description":"endpoint"
+                    },
+                    "sitedescription":{
+                      "type":"string",
+                      "description":"sitedescription"
```
Comment (sriyen-msft) [inline custom-connectors/My Acclaro/apiDefinition.swagger.json:59]: @msanz-acclaro All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: custom-connectors/My Acclaro/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#f0f0f0",
+    "capabilities": [],
+    "publisher": "Mariano Sanz"
```
Comment (sriyen-msft) [inline custom-connectors/My Acclaro/apiProperties.json:20]: @msanz-acclaro Please add `"stackOwner"` property.

Docs - [Add Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)

Comment (sriyen-msft) [inline custom-connectors/My Acclaro/apiProperties.json:20]: @msanz-acclaro For certified connectors, the publisher value should be company or organization name.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: custom-connectors/My Acclaro/apiDefinition.swagger.json (hunk 2 above)
Comment (msanz-acclaro) [reply custom-connectors/My Acclaro/apiDefinition.swagger.json]: this has been addressed

File: custom-connectors/My Acclaro/apiProperties.json (hunk 3 above)
Comment (msanz-acclaro) [reply custom-connectors/My Acclaro/apiProperties.json]: this has been addressed

Comment (msanz-acclaro) [reply custom-connectors/My Acclaro/apiProperties.json]: this has been addressed

File: custom-connectors/My Acclaro/README.md (hunk 1 above)
Comment (msanz-acclaro) [reply custom-connectors/My Acclaro/README.md]: @sriyen-msft moved from "custom-connectors" to "certified-connectors" as the intention is to get the connector certified. **Please accept my apologies**, the [documentation at MS](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission) is a bit misleading on this topic.

Comment (msanz-acclaro) [reply custom-connectors/My Acclaro/README.md]: moved to certified-connectors

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**